### ISSUE TITLE
How use follow?

### ISSUE THREAD

**Issue by kkaabbaa** [dherges/ng-hal#50]
I used this traverson before.
But since it does not support @angular/http I chose this library

Question:
Can I use follow like in traverson?

Here is an example code

```
traverson
     .from (path)
     .jsonHal()
     .follow(links)
```

Where 
`let links = 'memberinfo'` - exampe 'memberinfo'
or 
`let links = ['events', 'events[profileId:' + profileId + ']'];`

**Comment by dherges** [comment]
No, it works all-together different. In the current versions it works like [mergeMap in rxjs](https://www.learnrxjs.io/operators/transformation/mergemap.html) 


However, I think I will drop this behaviour of `follow` in the future.
